docs/content/users/install/ddev-installation.md

```diff
@@ -45,6 +45,15 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
 
     ## Linux
 
+    ### Locally-trusted certificate with mkcert
+
+    Modern browsers require valid certificates, which mkcert can create. Firefox users [may get prompted](https://github.com/FiloSottile/mkcert/issues/50) for `Password or Pin for "NSS Certificate DB` which is the Firefox Master pin. Install mkcert, and then run this:
```

Comment:
Reading the issue you linked to, looks like this is missing a quote, “master” is not a proper noun that needs to be capitalized, and it would be most helpful to refer to it as a “password” instead of re-using the word “pin” from the description. Also worth adding a link for “install mkcert”:

```suggestion
    Modern browsers require valid certificates, which mkcert can create. Firefox users [may get prompted](https://github.com/FiloSottile/mkcert/issues/50) for `Password or Pin for "NSS Certificate DB"` which is the Firefox master password. [Install mkcert](https://github.com/FiloSottile/mkcert#installation), and then run this:
```